main.go

```diff
@@ -42,11 +43,15 @@ func main() {
 	var zipper = tile.NewZipper()
 	var releaseCreator = winfsinjector.ReleaseCreator{}
 
-	wd, err := ioutil.TempDir("", "")
+	wd, err := os.MkdirTemp("", "")
 	if err != nil {
 		log.Fatal(err)
 	}
-	defer os.RemoveAll(wd)
+
+	fmt.Fprintf(os.Stdout, "tile extraction directory: %s\n", wd)
+	if !arguments.PreserveExtracted {
+		defer os.RemoveAll(wd)
+	}
```

Comment:
To keep the logs cleaner, let's only log the extraction directory when `--preserve-extracted=true`.

Reply:
Sounds good.  I was trying to have only one `if` statement to keep the code marginally cleaner, but I can see where logs would be the better priority.